jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -72,20 +99,34 @@ ecma_builtin_global_object_eval (ecma_value_t this_arg, /**< this argument */
     parse_opts |= ECMA_PARSE_STRICT_MODE;
   }
 
-  if (!ecma_is_value_string (x))
-  {
-    /* step 1 */
-    ret_value = ecma_copy_value (x);
-  }
-  else
+  /* steps 2 to 8 */
+  return ecma_op_eval (ecma_get_string_from_value (x), parse_opts);
+} /* ecma_builtin_global_object_eval */
+
+/**
+ * Helper function for trailing leading whitespaces
```

Comment:
I'm guessing you've meant 'trimming' instead of 'trailing'.

Reply:
Indeed. Thanks!